smith.lic

```diff
@@ -26,6 +26,8 @@ class Smith
       ]
     ]
 
+_respond "<pushBold/>Mahtra's test version<popBold/>"
```

Comment:
I assume this should be removed?

Reply:
Yes indeed. Sorry about that. I can fix it in about 6 hours.